test/pipeline_integration_test.exs

```diff
@@ -5,39 +5,59 @@ defmodule Membrane.VideoCompositor.PipelineTest do
 
   alias Membrane.RawVideo
   alias Membrane.Testing.Pipeline, as: TestingPipeline
+  alias Membrane.VideoCompositor.Test.Utility, as: TestingUtility
 
-  describe "Checks h264 pipeline on " do
+  describe "Checks h264 nx pipeline on " do
     @describetag :tmp_dir
 
-    test "2s 720p 30fps video", ctx do
-      input_paths = %{
-        first_h264_video_path: "./test/fixtures/short_videos/input_2s_720p.h264",
-        second_h264_video_path: "./test/fixtures/short_videos/input_2s_720p.h264"
+    test "2s 720p 30fps video", %{tmp_dir: tmp_dir} do
+      video_caps = %RawVideo{
+        width: 1280,
+        height: 720,
+        framerate: {30, 1},
+        pixel_format: :I420,
+        aligned: nil
       }
```

Comment:
If you make 1080p and 720p video caps a module attributes, tests could have only one line, something like:
```
test "1s 1080p 30fps video", %{tmp_dir: tmp_dir} do
  test_h264_pipeline(@raw_video_hd_caps, 1, :nx, "short_videos", tmp_dir)
end
```

Reply:
I'll change this in PR#9